network-grpc/proto/node.proto

```diff
@@ -111,6 +111,9 @@ service Node {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
 
+  // A one-off gossip exchange.
+  rpc ExchangeGossip(Gossip) returns (Gossip);
```

Comment:
Hmm no this was not the direction we wanted to go. Here we don't even know about gossips but about other peers. @vincenthz was looking into this too I believe.